certbot/main.py

```diff
@@ -382,6 +382,28 @@ def register(config, unused_plugins):
     # exist or not.
     account_storage = account.AccountFileStorage(config)
     accounts = account_storage.find_all()
+    reporter_util = zope.component.getUtility(interfaces.IReporter)
+    add_msg = lambda m: reporter_util.add_message(m, reporter_util.MEDIUM_PRIORITY)
+
+    if config.deactivate:
+        if len(accounts) == 0:
+            add_msg("Could not find existing account to deactivate.")
+        else:
+            yesno = zope.component.getUtility(interfaces.IDisplay).yesno
+            prompt = ("Are you SURE you would like to irrevocably deactivate "
+                      "your account? You will lose access to the domains "
+                      "associated with it.")
```

Comment:
I think the sentence "You will lose access to the domains associated with it." is a bit ambiguous and overstates the problem a bit. Instead, I think you can mention some of the other concerns from the help text for `--deactivate` and the comments I posted on that.